includes/syslog.php

```diff
@@ -57,8 +59,8 @@ function process_syslog($entry, $update)
     }
 
     $entry['host'] = preg_replace("/^::ffff:/", "", $entry['host']);
-    if (array_key_exists($entry['host'], $config['syslog_xlate'])) {
-        $entry['host'] = $config['syslog_xlate'][$entry['host']];
+    if (array_key_exists($entry['host'], $syslog_xlate)) {
```

Comment:
Close, try this:
```php
if (Config::has('syslog_xlate.' . $entry['host'])) {
    $entry['host'] = Config::get('syslog_xlate.' . $entry['host']);
}
```
or just

```php
if ($new_host = Config::get('syslog_xlate.' . $entry['host'])) {
    $entry['host'] = $new_host;
}
```

Reply:
Oh, those look much cleaner. I like the second one. I'll be submitting the changes soon.